### PR TITLE
Refactor, add .gitignore and script.js

### PR DESCRIPTION
**Pull request by Rohitrajak1807**
@deepakgupta191199 
Ignoring IDE configs,
refactor script
add label for text input

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,20 +7,16 @@
...
+    </form>
+    <div id="list">
 
-        </div>
     </div>
-    <script>
-        function enterTask(e){
-            e.preventDefault();
-            console.log(e)
-        }
```
Comment (deepakgupta191199) [inline index.html:23]: Why did you remove this??

Comment (Rohitrajak1807) [reply index.html:23]: > Why did you remove this??

Just for fun 🤣
